sword2/atom_objects.py

```diff
@@ -64,7 +64,7 @@ def _from_element(self, e):
         """ Load the `Category`'s internal attributes using the information within an `etree.SubElement`
         
         """
-        for item in e.attrib.keys():
+        for item in list(e.attrib.keys()):
```

Comment:
```suggestion
        for item in e.attrib:
```
As dict is by default calling dict.keys() which returns iterator in python 3 this will be sufficient (on every for loop where you are forcing list(dict.keys())

Reply:
OK, as this is not a dict but dom element attributes you probably want to call .keys() here, still forcing it to be a list is unnecessary and memory inefficient.

Reply:
Right, `list` conversions are unnecessary. I am removing them. `e.attrib` is a dom element but appearantly it is represented as a `dict` in the library. `for item in e.attrib` doesn't work. It should be either `e.attrib.keys()` or `e.attrib.items()`. I keep them as .keys() since we don't need values here.